relay-common/src/lib.rs

```diff
@@ -23,3 +23,17 @@ pub use crate::glob::*;
 pub use crate::macros::*;
 pub use crate::project::*;
 pub use crate::time::*;
+
+/// Validates a metric name. This is the statsd name, i.e. without type or unit.
+///
+/// Metric names cannot be empty, must begin with a letter and can consist of ASCII alphanumerics,
+/// underscores, slashes and periods.
+pub fn is_valid_metric_name(name: &str) -> bool {
+    let mut iter = name.as_bytes().iter();
+    if let Some(first_byte) = iter.next() {
+        if first_byte.is_ascii_alphabetic() {
+            return iter.all(|b| b.is_ascii_alphanumeric() || matches!(b, b'.' | b'_' | b'/'));
+        }
+    }
+    false
+}
```

Comment:
simply moved to `relay-common` as suggested, however I don't know where in relay-common to put it. is `lib.rs` fine? should I make a new module?

Reply:
Not strongly opinionated here, but I'd create a `metrics.rs` and put it there.

Reply:
ok changed to your suggesiton